lazyflow/utility/helpers.py

```diff
@@ -115,3 +118,12 @@ def get_default_axisordering(shape: Tuple[int, ...]) -> str:
         return axisorders[ndim]
     except KeyError:
         raise ValueError(f"cannot infer axis order for {ndim}-dimensional shape")
+
+
+def shapeprod(shape: Iterable[numbers.Integral]) -> int:
```

Comment:
Python 3.8 and later has [`math.prod`](https://docs.python.org/3/library/math.html#math.prod), so maybe it'd be better to add `lazyflow.future.math`, import it as `from lazyflow.future import math`, and `math.prod(shape)`? Later, we can replace `from lazyflow.future import math` with just `import math`.

Reply:
I agree that it would be nice, however, we'd still need to convert to int: 
Something like `shape = numpy.array((10**31, 100), dtype="uint32")` will resort in an overflow error...

Reply:
instead of `reduce(mul,...)` we could use `itertools.prod`...

Reply:
Yes, you are correct, I erroneously assumed that `math.prod` would convert elements to int...

Reply:
In this case, how about renaming `shapeprod` to `approd`, `bignumprod`, or something similar?

Reply:
hm good point. `shapeprod` is not very informative... how about `bigintprod` as yet another alternative @emilmelnikov? :)

Reply:
`bigintprod` is a perfect name!